conf/test.config

```
@@ -23,7 +23,8 @@ params {
     max_memory = '6.GB'
     max_time   = '6.h'
 
-    input  = "https://raw.githubusercontent.com/nf-core/test-datasets/pixelator/samplesheet/samplesheet.csv"
+    // Input data
+    input          = params.pipelines_testdata_base_path + 'pixelator/samplesheet/samplesheet.csv'
     input_basedir  = "https://raw.githubusercontent.com/nf-core/test-datasets/pixelator/testdata"
```

```suggestion
    input_basedir  = params.pipelines_testdata_base_path + 'pixelator/testdata'
```

You should be able to do something like that (untested)